app/graphql/types/query_type.rb

```diff
@@ -38,5 +38,13 @@ def search_materials(search:)
     def search_locations(materialId:, location:)
       RecycleFacade.search_locations(materialId, location)
     end
+
+    field :materials, [Types::MaterialInfoType], null: false
+
+    def materials
+      Rails.cache.fetch("material-list", expires_in: 7.days) do
```

Comment:
Cacheing is a great feature so that we won't have to put in a network request every time the page loads.